static/js/components/email/EmailCompositionDialog.js

```diff
@@ -93,6 +108,7 @@ export default class EmailCompositionDialog extends React.Component {
       onRequestClose={closeAndClearEmailComposer}
     >
       <div className="email-composition-contents">
+        { this.renderExtraUI() }
```

Comment:
I don't think you'll need the helper function, you could just do something like:

```js
{ R.isNil(extraUI) ? null : <extraUI {...this.props} /> }
```